exercicioLogica16/src/main/java/org/example/Pessoa.java

```diff
@@ -0,0 +1,19 @@
+package org.example;
+
+import java.util.ArrayList;
+
+
+public class Pessoa {
+
+    private ArrayList<String> membros = new ArrayList<>();
```

Comment:
não é necessário membros (nome pessoa?) ou docBiblioteca ser uma lista, nem ser ser inicializado na declaração do atributo.

Pessoa poderia ter só nome e docBiblioteca e no construtor utilizado na criação de uma nova Pessoa, como na classe Main, receber o nome e docBiblioteca

ex:
public Pessoa(String membros, Integer docBiblioteca) {
        this.membros = membros;
        this.docBiblioteca = docBiblioteca;
}

main:
Pessoa pessoa = new Pessoa("Jon", 123);